src/uct/rocm/copy/rocm_copy_iface.h

```diff
@@ -15,10 +15,16 @@ typedef uint64_t uct_rocm_copy_iface_addr_t;
 typedef struct uct_rocm_copy_iface {
     uct_base_iface_t super;
     uct_rocm_copy_iface_addr_t id;
+    struct {
+        size_t d2h_thresh;
```

Comment:
could you pls align field definitions in the structure (in the same way as it done for the structure below)?